src/main/java/frc/robot/RobotContainer.java

```diff
@@ -11,7 +11,6 @@
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
-import frc.robot.subsystems.ExampleSubsystem.ExampleSubsystem;
```

Comment:
This is good, but not a part of this PR. Please open another PR where you'll delete the example files.